source/transfer_list.rst

```diff
@@ -181,8 +221,9 @@ Inputs:
    equal to the size of the total area reserved for the TL (if known). If not,
    abort (TL is corrupted).
 
-#. *(optional)* Check that the sum of `tl.size` bytes starting at `tl_base_addr`
-   minus `tl.checksum` is equal to `tl.checksum`. If not, abort (TL is corrupted).
+#. *(optional)* If `has_checksum`, check that the xor of `tl.size` bytes
+   starting at `tl_base_addr` minus `tl.checksum` is equal to `tl.checksum`. If
```

Comment:
I think the "minus `tl.checksum`" isn't quite correct now anymore, that would also be an XOR, but if you write it like that I'm worried it might be a bit hard to read.

I think with XOR we should be able to just use the simpler alternative "check that the XOR of `tl.size` bytes starting at `tl_base_addr` is `0x0`" (because the value already in the checksum field should exactly cancel out the XOR of everything else excluding the checksum field), right?

Reply:
OK done